src/signals/incident/components/form/Header/__snapshots__/index.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Form component <Header /> error messages should render custom error 1`] = `
 <div
-  className="header header--invalid"
+  className=" header header--invalid"
```

Comment:
I understand this is a snapshot file however the space should not be there in the first place?